### ISSUE TITLE
The Markdown Highlight operator is not supported

### ISSUE THREAD

**Issue by farid98** [contentlayerdev/contentlayer#567]
If I have markdown like this:

`==Testing==
`

It should resolve into the HTML <mark> tag, i.e.

`<mark>Testing</mark>
`

which Tailwind prose plugin handles correctly and highlights in yellow by default.

Currently, it just shows up as ==Testing== i.e. it is not recognized a a valid Markdown modifier

Im aware that this tag is not supported by all markdown editors, but it is in fact a very handy one and very useful in blogposts. For instance the Bear markdown editor supports this. Please consider adding support

**Comment by johnpaulkiser** [comment]
contentlayer uses remark to render markdown to html. [Here](https://github.com/remarkjs/remark/issues/659) is the relevant issue on their repo about this topic.

TLDR: Remark has left syntax extensions (like highlighter) up to plugins. [remark-mark-plus](https://github.com/twardoch/remark-mark-plus) (I haven't personally used this) is one such plugin that handles this.
